instructor/function_calls.py

```diff
@@ -237,6 +239,14 @@ def from_response(
                 context=validation_context,
                 strict=strict,
             )
+        elif mode == Mode.MD_JSON:
+            pattern = r'```(?:json)?(.*?)```'
+            json_content = re.findall(pattern, message.content, re.DOTALL)[0].strip()
+            return cls.model_validate_json(
+                json_content.strip(),
```

Comment:
it should just be content. and the ````(?:json)?(.*?)```` should be part of the decoding by starting the message with ```json